takwimu/context_processors.py

```diff
@@ -18,42 +18,52 @@ def takwimu_countries(request):
             {
                 'name': 'Burkina Faso',
                 'name_short': 'Burkina Faso',
+                'name_slug': 'burkina-faso',
```

Comment:
I think we should just call this `slug` and use the `slugify` function to handle this automatically since it even handles unicode characters like `è` to `e` in a consistent manner. i.e.

```python
...
from django.utils.text import slugify

...

   'name_short', 'Burkina Faso',
   'slug', slugify('Burkina Faso'),

Reply:
@kilemensi is the `slugify` really necessary? I mean we're already writing out the name as a constant so we could manually handle unicode since there are only 10 countries

Reply:
No, it's totally not necessary @phillipahereza. If you will ensure no copy/paste mistakes it's good enough for me.